src/ui/zcl_abapgit_gui_page_sett_repo.clas.abap

```diff
@@ -121,6 +122,10 @@ CLASS zcl_abapgit_gui_page_sett_repo IMPLEMENTATION.
       iv_label       = 'Main Language'
       iv_hint        = 'Main language of repository (cannot be changed)'
       iv_readonly    = abap_true
+    )->text(
+      iv_name        = c_id-i18n_langs
+      iv_label       = 'LXE langs (experimental)'
```

Comment:
LXE > "Translations (experimental)"

"master language" > "main language"

I see you sneaked Ukranian in there 😀

Reply:
![image](https://user-images.githubusercontent.com/15635498/104857438-4a48a600-5921-11eb-90bf-908c81bdda29.png)
 What about this ?

Reply:
Sure.